crates/flux-tests/tests/lib/rset.rs

```diff
@@ -0,0 +1,41 @@
+#![allow(dead_code)]
+#![flux::defs {
+    fn set_add<T>(x: int, s: Set<T>) -> Set<T> { set_union(set_singleton(x), s) }
```

Comment:
I was very confused about how this could possibly be working, but I think it is not being executed. Maybe move it to todo.